app/registries/foreman/plugin/assets.rb

```diff
@@ -30,8 +30,13 @@ def webpack_manifest_path
         File.file?(manifest_path) ? manifest_path : nil
       end
 
+      def webpack_remote_entry
+        remote_entry_path = File.join(path, 'public', 'webpack', id.to_s, "#{id}_remoteEntry.js")
+        File.file?(remote_entry_path) ? remote_entry_path : nil
+      end
+
       def uses_webpack?
-        path && (File.file?(File.join(path, 'webpack', 'index.js')) || webpack_manifest_path.present?)
+        path && (File.file?(File.join(path, 'webpack', 'index.js')) || webpack_manifest_path.present? || webpack_remote_entry.present?)
```

Comment:
arguably, `webpack_manifest_path.present?` can be dropped now
also, this method hits the disk on every page load, we should cache the result in memory (but scope creep!)

Reply:
you mean "hits the disk" part?